src/game_initialization/multiplayer.cpp

```diff
@@ -536,16 +540,21 @@ bool mp_manager::enter_lobby_mode()
 
 		int dlg_retval = 0;
 		int dlg_joined_game_id = 0;
+		std::string preset_scenario = "";
 		{
 			gui2::dialogs::mp_lobby dlg(lobby_info, *connection, dlg_joined_game_id);
 			dlg.show();
 			dlg_retval = dlg.get_retval();
+			preset_scenario = dlg.queue_game_scenario_id();
```

Comment:
This should be a preset ID but I guess that doesn‘t currently fit.

Reply:
I don't understand this comment.

Reply:
The point is to use data from a game preset which cannot be identified by a scenario id since you can have multiple presets for the same scenario.

Reply:
I have no goal of setting up multiple presets for the same scenario though.